iphone/Classes/TiContactsPerson.m

```diff
@@ -618,7 +611,7 @@ -(id)valueForUndefinedKey:(NSString *)key
 #if IS_XCODE_7
 	if ([TiUtils isIOS9OrGreater]) {
 		//birthday property managed seperately
-		if ([key isEqualToString:@"birthday"]) {
+		if ([key isEqualToString:@"birthday"] && [person isKeyAvailable:CNContactBirthdayKey]) {
```

Comment:
Yes thats true. But the behaviour is inconsistent. It works well in iPhone 6 (9.1) simulator only! Not sure about device. Anyways I will add this note to docs.